docs/comparison.md

```diff
@@ -0,0 +1,141 @@
+extensive, but we attempt to lean on good and well funded maintenance at Formidable, active
+investment, and good documentation. So you may have to evaluate whether you can get any questions
+answered quickly enough on [GitHub Discussions](https://github.com/FormidableLabs/urql/discussions)
+for instance.
+
+### Caching and State
+
+| ---                                                     | urql                                                                  | Apollo             | Relay                                          |
+| ------------------------------------------------------- | --------------------------------------------------------------------- | ------------------ | ---------------------------------------------- |
+| Caching Strategy                                        | Document Caching, Normalized Caching with `@urql/exchange-graphcache` | Normalized Caching | Normalized Caching (schema restrictions apply) |
+| Added Bundle Size                                       | +5.9kB (with Graphcache)                                              | +0 (default)       | +0 (default)                                   |
+| Automatic Garbage Collection                            | ✅                                                                    | ✅                 | ✅                                             |
+| Local State Management                                  | 🛑                                                                    | ✅                 | ✅                                             |
+| Pagination Support                                      | 🔶                                                                    | 🔶                 | ✅                                             |
+| Optimistic Updates                                      | ✅                                                                    | ✅                 | ✅                                             |
+| Local Updates                                           | ✅                                                                    | ✅                 | ✅                                             |
+| Out-of-band Cache Updates                               | 🛑 (stays true to server data)                                        | ✅                 | 🛑                                             |
+| Local Resolvers and Redirects                           | ✅                                                                    | ✅                 | 🛑                                             |
+| Complex Resolvers (nested non-normalized return values) | ✅                                                                    | 🛑                 | 🛑                                             |
+| Commutativity Guarantees                                | ✅                                                                    | 🛑                 | 🛑                                             |
```

Comment:
Could link to under-the-hood to explain this

Reply:
It's explained in the explanation after the table